lib/rspec/core/reporter.rb

```diff
@@ -1,7 +1,17 @@
+require 'set'
 module RSpec::Core
```

Comment:
I think it looks weird to not have a blank line between the require and the module def, but since you'll be removing the set require in another PR soon feel free to merge with this as-is :).